test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1317,7 +1332,7 @@ func newContainerRuntimeConfig(name string, profile *performancev2.PerformancePr
 				MatchLabels: profilecomponent.GetMachineConfigPoolSelector(profile, profileMCP),
 			},
 			ContainerRuntimeConfig: &machineconfigv1.ContainerRuntimeConfiguration{
-				DefaultRuntime: machineconfigv1.ContainerRuntimeDefaultRuntimeCrun,
+				DefaultRuntime: machineconfigv1.ContainerRuntimeDefaultRuntimeRunc,
```

Comment:
why default changed here to runc?

Reply:
Since the workflow of this e2e is as follows:
First we test without any ctrcfg in the system
Second, we apply a ctrcfg with a different runtime than what's already in the system (on 4.18 it is crun) as we do want to test a change and not override the runtime with the same runtime.

Reply:
Ack. Got confused by the name